standards/README.md

```diff
@@ -2,6 +2,18 @@
 
 This folder contains all current standards for software development.
 
+Our standards are intended to:
+
+- Make the produced code more easily read and understood
+- Make the code more easily maintainable
+- Make the code more reliable
+
+## Versioning
+
+Our standards are updated on an ongoing basis. Anything using these standards should therefore refer to these standards not only by name, for example **C# coding standards**, but also by date, for example **as at 2020-01-08**.
+
+Whilst we always endeavour to keep code consistent with the latest standards, not all projects are active. This means as standards change some projects will no longer meet them. Having a reference to when the standard was last applied, will help when determining the effort to update a project
```

Comment:
```suggestion
Whilst we always endeavour to keep code consistent with the latest standards, not all projects are active. This means as standards change some projects will no longer meet them. Having a reference to when the standard was last applied, will help when determining the effort to update a project.
```

I also wonder if it would be useful to say where this reference should be? Project README?